src/jit/lsraarmarch.cpp

```diff
@@ -711,6 +711,12 @@ void Lowering::TreeNodeInfoInitPutArgStk(GenTreePutArgStk* argNode, fgArgTabEntr
             MakeSrcContained(argNode, putArgChild);
         }
     }
+    else if (putArgChild->TypeGet() == TYP_DOUBLE) {
```

Comment:
I think this `else if` block should be wrapped in `#ifdef _TARGET_ARM_` since this file is for both ARM and ARM64.